dartagnan/src/main/java/com/dat3m/dartagnan/verification/VerificationTask.java

```diff
@@ -44,6 +48,16 @@
 @Options
 public class VerificationTask {
 
+    @Option(
+            name = OptionNames.PROPERTY,
+            description = "The property to check for: reachability (default), liveness, races.",
+            toUppercase=true)
+    private EnumSet<Property> property = EnumSet.of(Property.getDefault());
+
+	public EnumSet<Property> getProperty() { return property; }
+
+    // =====================================================================
+
```

Comment:
`BaseOptions` already has Property defined. So just add a `setProperty` method to the `VerificationTaskBuilder` and let the constructor of `VerificationTask` take the property as an input parameter. This avoids the duplication of having two identical property options in two different objects.